test/config.test.ts

```diff
@@ -1,15 +1,19 @@
+import fsCb from 'fs';
 import path from 'path';
-import getConfigs from './config';
+import Config from '../src/config';
+
+const fs = fsCb.promises;
 
 describe('getConfig()', () => {
-  it('reads pipeline files and returns an array of config files - empty', async () => {
-    process.chdir(__dirname);
```

Comment:
Once tests move back to the `tests/` directory, any that are at the top level, like this one, are now underneath the `projects/` directory (which they see, because we do recursive traversal looking for pipeline components these days)